packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -27,6 +27,8 @@ nls.config({
 })();
 const localize: nls.LocalizeFunc = nls.loadMessageBundle();
 
+const HTTP_UNAUTHORIZED = 401;
```

Comment:
We could also import `RestConstants.HTTP_STATUS_401` from Imperative to use that constant 🤷  https://github.com/zowe/imperative/blob/ca981d476b7a97fa6129de523c63e1a58000c139/packages/rest/src/client/RestConstants.ts#L66